x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -161,6 +167,83 @@ export const getMetadataListRequestHandler = function (
   };
 };
 
+export function getMetadataListRequestHandlerV2(
```

Comment:
will remove the V2 naming when we get rid of the POST API.

Reply:
most of the code is purposely duplicated so that the POST API can be easily removed.